packages/contracts/contracts/interfaces/IERC20Extended.sol

```diff
@@ -0,0 +1,95 @@
+pragma solidity ^0.8.4;
```

Comment:
I think this interface is already part of the openzeppelin contract package we have right?

Reply:
No. I don't think there is an interface with DECIAML parameters.

Reply:
You're right, that's very silly.